docker-compose.yaml

```diff
@@ -2,14 +2,14 @@
 services:
   node:
     container_name: sample_validator
-    image: "alloranetwork/allora-chain:v0.4.0-upgrader" # docker image without cosmovisor. use vx.x.x-upgrader for upgrade image
+    image: "alloranetwork/allora-chain:v0.4.1-docker-upgrade" # docker image without cosmovisor. use vx.x.x-docker-upgrade for upgrade image
```

Comment:
needs to be v0.5.0

Reply:
https://github.com/allora-network/allora-chain/pull/594